app/MyApp/index.html

```diff
@@ -476,7 +476,7 @@
                 <tr class="imageRow">
                     <td rowspan="2" class="imageData">
                         <!--<img id="header-image" src="img/header_slice.png" /> -->
-                        <img src="img/OLE_Logo.png" width="108px" height="108px" id="imgurl" style="z-index:1; border:2px solid white;border-radius:55px;height: 105px; width: 105px;"/>
+                        <img src="img/OLE_Logo.png" width="108px" height="108px" id="imgurl" style="z-index:1; border:2px solid white;border-radius:55px;height: 105px; width: 105px; background: white"/>
```

Comment:
thought we had already a pic with white background ...
anyhow transparent with white background is an awesome hack ;)